src/webkit/WebKit_Startup_002.test.js

```diff
@@ -0,0 +1,45 @@
+import {
+  pluginDeactivate,
+  checkIfProcessIsRunning,
+  restartFramework,
+  webKitBrowserOps,
+  stopWPEFramework,
+  startFramework,
+} from '../commonMethods/commonFunctions'
+import constants from '../commonMethods/constants'
+
+export default {
+  title: 'WPEWebkit shutdown of Framework robustness test',
+  description: 'Starts WPEWebkit and stops Framework. Checks if everything is shutdown correctly',
+  repeat: 30,
+  setup() {
+    return this.$sequence([
+      () => pluginDeactivate.call(this, constants.youTubePlugin),
+      () => pluginDeactivate.call(this, constants.netFlixPlugin),
+    ])
+  },
+  teardown: restartFramework,
+  steps: [
+    {
+      description: 'Activate WebKit Plugin and check whether it is resumed correctly',
+      test: webKitBrowserOps,
+      assert: 'resumed',
+    },
+    {
+      description: 'Stop Framework',
+      test: stopWPEFramework,
+      assert: true,
+    },
+    {
+      description: 'Check if WebKitImplementation rpcprocess is gone',
+      test: checkIfProcessIsRunning,
+      params: constants.webKitImplementation,
+      assert: false,
+    },
+    {
+      description: 'Start Framework',
+      test: startFramework,
+      assert: true,
+    },
```

Comment:
I know this is unreleated to the test case convert, but looking at it we likely want to add something to check if WPEFramework came up successfully. Like a few calls to the controller to valdiate it is behaving as expected before we rerun the test.

Reply:
Added validation to check whether the controller is responding and the response from controller is object or not.